tukaan/widgets/tooltip.py

```diff
@@ -1,4 +1,5 @@
 from tukaan import Label
+from tukaan._tcl import Tcl
 
```

Comment:
Just noticed, `isort` had an issue with this line. It likes two blank lines between imports and the start of the actual code, that was its problem.